lib/index.js

```diff
@@ -36,20 +41,88 @@ class AttributeList {
 
     this.el.setAttribute(this.attributeName, this.tokenList.join(" "))
   }
-}
 
-export function $al(el) {
-  function toCamelCase(str){
-    return str.replace(/^([A-Z])|[\s-_]+(\w)/g, (match, p1, p2, offset) =>  p2 ? p2.toUpperCase() : p1.toLowerCase());
+  remove(attr){
+    var parsed = AttributeList.parseAttributeFakeTokenList(arguments, 'add');
+    parsed.forEach(token => {
+      let pos = this.tokenList.indexOf(token)
+      if (pos > -1){
+        this.tokenList.splice(pos, 1)
+      }
+    })
+
+    this.el.setAttribute(this.attributeName, this.tokenList.join(" "))
+  }
+
+  toggle(attr){
+    let pos = this.tokenList.indexOf(attr)
+    if (pos > -1){
+      this.remove(attr)
+    } else {
+      this.add(attr)
+    }
+    this.el.setAttribute(this.attributeName, this.tokenList.join(" "))
   }
+}
+
+
+// Helper
+function toCamelCase(str){
+  return str.replace(/^([A-Z])|[\s-_]+(\w)/g, (match, p1, p2, offset) =>  p2 ? p2.toUpperCase() : p1.toLowerCase());
+}
+
+
+/*
+ * @name $attributeList
+ * @description AttributeList Wrapper Mode, se like _loadash or jquery
+ *
+ * @param {HTMLElement|String} el — target element
+ */
+export function $attributeList(el) {
+  // Sidebar element query if there's no one, throw error.
+  let _el = ('string' === typeof el ) ? document.querySelector(el) : el;
+  if( 'undefined' === typeof el )
+    throw new Error("There is no specific sidebar element.");
 
-  var al = el.getAttributeNames();
-  let obj = {};
+  // todo: should we remove other attributes from manipulation and only leave data-attributes??
+  let al = _el.getAttributeNames().filter(a => a !== "id" && a !== "class");
+  let alObj = {};
 
   al.forEach(attr =>  {
-    var camelCased = toCamelCase(attr);
-    obj[camelCased] = new AttributeList(el, attr)
+    let camelCased = toCamelCase(attr.replace("data-", ""));
+    alObj[camelCased] = new AttributeList(_el, attr)
   });
 
-  return obj
+  return alObj
 }
+
+
+/*
+ * [DANGEROUS]
+ *
+ * @function dangerouslyExtendElementPrototype
+ *
+ * @description Dangerous because i'm a designer and i don't have
+ * enough javaScript (programming) nerdery to fully
+ * understand how bad is what i'm doing here
+ */
+export function dangerouslyExtendElementPrototype(){
+  Object.defineProperty(Element.prototype, "yolo", {
```

Comment:
I commited `yolo` instead of `attributeList`, that's why you shouldn't copy paste code. 🤦‍♂️